src/odoc/page-index.mld

```diff
@@ -0,0 +1,252 @@
+{1:top odoc}
+
+The documentation compiler for OCaml, Reason, and BuckleScript.
```

Comment:
Does BuckleScript belong here? You could as well say dune in that case. Any tool can use odoc really, I'd keep just "OCaml and Reason".

Reply:
Agreed.

Reply:
As long as we convey properly that this can be used with BuckleScript compilation units, I'm up for rephrasing.

It's not obvious to everyone, particularly since Reason can be used with the OCaml toolchain.

Reply:
I think this can be explained later, i.e., the fact that odoc is designed to be integrated into existing build toolchains like BuckleScript and dune. A "documentation compiler" operates on syntax; BuckleScript is a compiler, that's why I find it confusing.

Also, ideally the end users shouldn't worry about how odoc works at all. That's why I think it's important to have a separate guide that describes the documentation features for documentation authors.